README.md

```diff
@@ -59,7 +59,7 @@ This plugin requires:
 |-------------------|----------------|
 | collectd          |     4.9+       |
 | Python plugin for collectd | (included with SignalFx collectd) |
-| Python            |     2.6+       |
+| Python            |     3+         |
```

Comment:
I believe we've got similar information in the integrations repo for several monitors. We should go in update those as well.